### ISSUE TITLE
Incorrect reporting that Listbox does not allow aria-orientation

### ISSUE THREAD

**Issue by SiTaggart** [dequelabs/axe-core#276]
When applying `aria-orientation` to a component with role `listbox`, aXe will report a critical violation: 

- `Ensures ARIA attributes are allowed for an element's role`
- Summary: `ARIA attribute is not allowed: aria-orientation="horizontal"`

Given that the Implicit value for the role has a default `aria-orientation` of `vertical` it would seem this is a bug. You should be able to at least provide another orientation value that is not the default, especially if your listbox goes from left to right.

http://w3c.github.io/aria/aria/aria.html#listbox

**Comment by AutoSponge** [comment]
Same for `tablist`. I think the problem is the use of [WAI-ARIA States and Properties Taxonomy](http://www.w3.org/TR/wai-aria/states_and_properties#state_prop_taxonomy) and [WAI-ARIA Definition of Roles](http://www.w3.org/TR/wai-aria/roles#role_definitions) in the [rules](https://dequeuniversity.com/rules/axe/2.1/aria-allowed-attr?application=axeAPI) instead of [WAI-ARIA 1.1](https://www.w3.org/TR/wai-aria-1.1/#tablist). There seem to be [conflicts](https://github.com/w3c/wcag/issues/259).

**Comment by marcysutton** [comment]
Fixed with #479.
